src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -10,17 +10,17 @@
 using OSPSuite.Core.Domain.Data;
 using OSPSuite.Core.Domain.Services;
 using OSPSuite.Core.Domain.UnitSystem;
-using OSPSuite.Core.Import;
 using OSPSuite.Core.Services;
 using OSPSuite.Utility.Extensions;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using OSPSuite.Infrastructure.Import.Core;
 using ColumnInfo = OSPSuite.Infrastructure.Import.Core.ColumnInfo;
 using OSPSuite.Infrastructure.Import.Services;
 using static OSPSuite.Core.Domain.Constants;
 using Command = OSPSuite.Assets.Command;
-using DimensionInfo = OSPSuite.Infrastructure.Import.Core;
```

Comment:
Still some old stuff here. In fact most of the code in this file seemed to be outdated. It was not using the default configuration etc...